rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.6.0-release-notes.adoc

```diff
@@ -6,14 +6,36 @@ include::partial$release-notes/rn-last-version-message.adoc[]
 
 == April 29, 2024
 
+//new action steps 
+* The new Submit Document to MuleSoft IDP action step enables you to send documents to IDP and process them with your published document actions. 
+* The new Retrieve Results from MuleSoft IDP action step enables you to retrieve the results of a document processed by IDP after using the Submit Document to MuleSoft IDP action step.
+* The new Extract Pages From a PDF File action step enables you to extract, split, and merge PDF files. 
```

Comment:
If these features are taken from the JTBD point of view, the sentences may read:
You can now retrieve the results of a document processed by IDP with the new Retrieve Results from MuleSoft IDP action step after using the Submit Document to MuleSoft IDP action step.